SITES-10835: Documentation (#26)
AdobeDocs/uix: src/pages/services/aem-cf-editor/api/header-menu/index.md

| onClick  | `callback(): void`                                                          |  ✔️ | A callback for a button `onClick` event                                                                       |

**Sub menu item API**

| Field    | Type                                                                        | Required | Description                                                                                                  |
|----------|-----------------------------------------------------------------------------| ------ |--------------------------------------------------------------------------------------------------------------|
| id       | `string`                                                                    | ✔️  | **Must be unique** across the current button sub menu                                                        |
| label    | `string`                                                                    | ✔️  | Button label that will be visible on UI                                                                      |

> | icon     | `string`                                                                    |    | An icon field accepts workflow icon code from @spectrum-icons library - https://spectrum.adobe.com/page/icons/ |

Reviewer: ```suggestion
| icon     | `string`                                                                    |    | Name of a [React-Spectrum workflow icon](https://react-spectrum.adobe.com/react-spectrum/workflow-icons.html#available-icons). |
```

| onClick  | `callback(): void`                                                          |  ✔️ | A callback for a button `onClick` event                                                                      |

# Extension Points API

> In our example, we used snippet:

Reviewer: ```suggestion
In our example, we used this snippet:
```

```js
const contentFragment = await guestConnection.host.contentFragment.getContentFragment();`
```

> [More details](../index.md) about it and the other Extension Points API.

Reviewer: ```suggestion
[More details](../index.md) about this and other Extension Points APIs.
```
